src/aspire/commands/denoise.py

```diff
@@ -67,10 +67,8 @@ def denoise(starfile_in, data_folder, starfile_out, pixel_size, max_rows, max_re
     var_noise = noise_estimator.estimate()
     # img_whitened = source.eval_filters(source.images())
     if denoise_method == 'CWF':
-        denoise_cov2d = DenoiserCov2D(source, basis, var_noise, ctf_info)
+        denoise_cov2d = DenoiserCov2D(source, basis, var_noise, ctf_info, starfile_out)
```

Comment:
Why are we providing `starfile_out` to `DenoiserCov2D`? We should be able to denoise images *without* saving them to a STAR file.

Reply:
This star file is an output for future analysis of other modules such as 2D classification. I think it might be useful to keep the similar format with Relion.

Reply:
Removed from the DenoiseCov2Dand used a new  `DenoisedImageSource` class for this in the new commit 23ca25c.